src/screens/collectibles-home/collectibles-home.tsx

```diff
@@ -104,7 +107,7 @@ export const CollectiblesHome = () => {
 
   const snapPoints = useMemo(
     () => [
-      windowHeight - (headerHeight + TAB_BAR_HEIGHT),
+      windowHeight - (headerHeight + TAB_BAR_HEIGHT + 4),
```

Comment:
what is 4? mb move it to const
and also, mb better formatSize(4)

Reply:
It's a primary fix, maybe it will be deleted in the next task